src/components/Dialog/__stories__/StoryComponents/StoryDialogContent.scss

```diff
@@ -1,5 +1,5 @@
 .story-dialog-content {
-  box-shadow: 0 4px 17px 6px rgba(0, 0, 0, 0.1);
+  box-shadow: 0 4px 17px 6px rgb(0 0 0 / 10%);
```

Comment:
not sure it's needed, I prefer the left syntax.

and seems like scss didn't know how to read it